### PR TITLE
79 user drilldown

### PR DESCRIPTION
**Pull request by natoverse**
This includes some basic accordion-like functionality for showing the repos underneath a user and vice-versa.
@bmathews @stopyoukid

### CODE REVIEW

File: app/client/src/modules/common/directives/user/user.js [hunk 1]
```diff
@@ -6,6 +6,24 @@ module.exports = /*@ngInject*/
...
                 user: '='
-            }
+            },
+            controllerAs: 'userCtrl',
+            bindToController: true,
+            controller: /*@ngInject*/
+                function (reposService) {
+                    // use this var to toggle a class using ng-class="{visible: ctrl.toggled}"
+                    // in the view to toggle a div where the repos will show up
+                    this.toggled = false;
```
Comment (bmathews) [inline app/client/src/modules/common/directives/user/user.js]: some left over comments that aren't needed

File: app/server/middleware/send.js [hunk 2]
```diff
@@ -4,6 +4,17 @@
...
+        let delay = req.query.delay;
+        if (delay > 0) {
+            setTimeout(() => {
+                next();
+            }, delay);
+        } else {
+            next();
+        }
+    },
+
```
Comment (bmathews) [inline app/server/middleware/send.js]: did you want this in this PR?

**Comment by bmathews** [comment]
Looks good!

File: app/server/middleware/send.js (hunk 2 above)
Comment (natoverse) [inline app/server/middleware/send.js]: Nope

**Pull request merged**